chef/cookbooks/crowbar/templates/default/crowbar-post-upgrade.sh.erb

```diff
@@ -19,6 +19,15 @@ fi
 
 <% end %>
 
+<% if @has_drbd %>
+
+if [[ -f /sbin/drbdadm ]] ; then
+    echo "Recreating DRBD metadata after the upgrade to new DRBD version..."
+    drbdadm -- --force create-md all
+fi
+
+<% end %>
+
```

Comment:
This is superfluous you already add exactly the same block in the previous commit.